tic-tac-toe-new-UI

```diff
@@ -2,14 +2,14 @@ def sum(a, b, c ):
     return a + b + c
 
 
-def printBoard(playerX, playerY):
+def printBoard(playerY, playerX):
```

Comment:
This should be the other way around - (playerX, playerY)

Reply:
noted